content/docs/command-reference/plots/index.md

```diff
@@ -96,7 +84,143 @@ names in the `train` array below:
 }
 ```
 
-## Plot templates (data series only)
+## Definining a plot
+
+In order to create visualizations, users need to provide the data and
+(optionally) configuration that will help customize the plot. DVC provides two
+ways to configure visualizations. Users can mark specific <abbr>stage</abbr>
+<abbr>outputs</abbr> as plot or define plot configuration inside `dvc.yaml`
+under `plots` key.
+
+### Standalone plots
+
+Plots defined in `dvc.yaml` under the `plots` key are especially useful when
+users want to compare data from different data sources residing on the same
+version of the project. For example, comparing training versus test results on
```

Comment:
```suggestion
Plots can be defined in `dvc.yaml` under the `plots` key. Unlike
[stage plots](#stage-plots), these are especially useful when
users want to compare data from different data sources residing on the same
version of the project. For example, comparing training versus test results on
```

It might be better to split this into two sentences. Explain what standalone plots are, then explain their unique benefit in the next sentence (and reference stage plots since it's not otherwise clear what you are comparing them to).